terasoluna-gfw-functionaltest-web/src/main/java/org/terasoluna/gfw/functionaltest/app/exceptionhandling/VariationExceptionLoggingFilter.java

```diff
@@ -0,0 +1,49 @@
+package org.terasoluna.gfw.functionaltest.app.exceptionhandling;
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+
+import org.terasoluna.gfw.web.exception.ExceptionLoggingFilter;
+
+public class VariationExceptionLoggingFilter extends ExceptionLoggingFilter {

```

Comment:
- [ ] I think that it's better to be with name as for example `LegacyDefaultExceptionLoggingFilter` or `ExceptionCodeExceptionLoggingFilter`. What do you think?